src/main/java/frc/robot/Constants.java

```diff
@@ -88,9 +88,10 @@
 		public static final Port kQueueColorSensorPort = Port.kTwo;
 		public static final Port kShooterColorSensorPort = Port.kThree;
 
-		public static final double kBottomShooterSpeedRPM = 2100;
-		public static final double kTopShooterSpeedRPM = 5300;
-
+//Motor and shooter are the same for the bottom flywheel, as there is a 1:1 ratio
```

Comment:
This comment doesn't make sense

Reply:
we are making a distinction between the actual speed of the flywheel and the speed of the motor shaft - they are different for the top flywheel because of the gear ratio but the same for the bottom, so we added the comment to avoid confusion about why the top has two values and the bottom doesn't.